js/mexc3.js

```diff
@@ -3094,8 +3094,15 @@ module.exports = class mexc3 extends Exchange {
         };
         let currency = undefined;
         if (code !== undefined) {
+            const rawNetwork = this.safeString (params, 'network');
```

Comment:
I think `network` might need to be recognized as unified network name too (not only exchange specific network-id), and that should be transformed into rawNetwork. i.e. if user calls `fetchDeposits ('USDT', ..,.., {network:'TRC20'})`  (or other unified names [mentioned here](https://discord.com/channels/690203284119617602/921046068555313202/968499931193155604)) then it should be transformed into `TRX` or whatever exchange specific chain-name is for mexc.  

--update--
It's a bit tricky, because mexc has different chain-names for same network, and maybe you might leave your implementation as is.

Reply:
@ttodua the thing is for most of the other cases mexc uses TRC20, so supposedly that would be the exchange-specific id, however for this method requires "TRX" so we don't have yet a safe way to convert unified network <-> exchange specific network

Reply:
> @ttodua the thing is for most of the other cases mexc uses TRC20, so supposedly that would be the exchange-specific id, however for this method requires "TRX" so we don't have yet a safe way to convert unified network <-> exchange specific network

@carlosmiei see how that is done in okx/huobi in withdraw()/fetchDepositAddres()